x/authz/simulation/operations.go

```diff
@@ -259,17 +259,9 @@ func SimulateMsgExec(ak authz.AccountKeeper, bk authz.BankKeeper, k keeper.Keepe
 			return simtypes.NoOpMsg(authz.ModuleName, TypeMsgExec, "not a send authorization"), nil, nil
 		}
 
-		if sendAuth.SpendLimit.IsAllLTE(coins) {
-			return simtypes.NoOpMsg(authz.ModuleName, TypeMsgExec, "over spend limit"), nil, nil
-		}
-
 		res, err := sendAuth.Accept(ctx, msg[0])
```

Comment:
this error isn't checked anymore, is that intentional?

Reply:
Yes, we can ignore err. I've updated it to be more clear.